app.py

```diff
@@ -306,7 +296,8 @@ def run_optimization(
     cost_table: list,
     previous_parameter_hash: str,
     cost_comparison: dict,
-) -> tuple[str, list, str, str, bool, str, str, dict, int, str, str, str, int, int]:
+    results_tab_classes: str,
+) -> tuple[str, list, str, str, bool, str, str, dict, int, str, str, str, int, int, str]:
```

Comment:
Not sure if this is dictated by Dash, but this rtype is extremely fragile. `namedtuple` would be much better here.

Reply:
Yeah I'm not a fan of this list.. I don't think Dash cares what we do with return types, but I haven't used namedtuples before, how do they work?

Reply:
Let me know if this is what you meant or if I'm confused 😄 https://github.com/dwave-examples/mvrp/pull/11/commits/95800df000f6977053e3aeaf888a098336a63126